docs/user/guides/best-practice/index.rst

```diff
@@ -0,0 +1,36 @@
+How-to guides: best practice
+============================
+
+Over the years,
+we have become familiar with a number of methods that work well and which we consider **best practice**.
+
+⏩️ :doc:`Best practices for linking to your documentation </automatic-redirects>`
```

Comment:
This is still really weird that this is the title of our automatic redirects page. Definitely something odd there.

Reply:
Do you want an explanation or are you saying that it'd be nice to have it fixed because it's getting too strange? :)

I've moved it now. There are more file names that are misleading, I think we need to deal with this separately.

Reply:
I think it's just debt we need to fix. I don't think the automatic redirects page should talk about linking to your docs, but instead mention automatic redirects?